plugins/woocommerce/includes/class-wc-auth.php

```diff
@@ -333,15 +333,15 @@ protected function auth_endpoint( $route ) {
 				 */
 
 				// Check if Jetpack is installed and activated.
-				if ( class_exists( 'Jetpack' ) && Jetpack::connection()->is_active() ) {
+				if ( class_exists( 'Jetpack' ) && Jetpack::connection()->has_connected_owner() ) {
 
 					// Check if the user is using the WordPress.com SSO.
 					if ( Jetpack::is_module_active( 'sso' ) ) {
```

Comment:
Thanks for the quick fix on this!

Since you're touching this part of the codebase, I wonder if it may be worth moving away from the `Jetpack` class as well, and instead rely on `\Automattic\Jetpack\Connection\Manager`'s `has_connected_owner` and `\Automattic\Jetpack\Modules`'s `is_active` method?